_posts/2022-02-28-war-in-ukraine.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "The War in Ukraine"
+author: [michal.kosmulski]
+tags: [tech, "soft skills", helping]
+---
+The attack on Ukraine has shocked the whole world, but in Poland it is perceived in an especially poignant way.
+Not only is Ukraine our direct neighbor (as are Russia and Belarus), but Poland’s difficult history allows us to
+sympathize with the Ukrainians even more. Many live in Poland, so it’s affecting people we know personally.
+
+While the fighting continues, here’s what [Allegro](https://allegro.tech) is doing, and what you can do, too, to help
+humanitarian efforts.
+
+* If you know any Ukrainians, your coworkers in particular, talk to them and show your support. Even minor gestures are
+  important. Ask directly what kind of help they might need rather than trying to guess.
+* Do not marginalize Russians and Belorussians. People are not the same as their governments, and often do not support
+  their states’ actions.
+* Work with your HR people and consider what kind of help you can offer as a company. Some examples would be extra days
+  off, legal and financial support for bringing families to safety, or offering psychological support.
+* On a personal level, consider supporting one of the trustworthy humanitarian organizations. You can support them
+  with cash or by donating necessary items which are being collected in most cities.
+* Ask if the company can [donate directly, make donations by others easier](https://spolecznosc.allegro.pl/t5/blog/wsparcie-dla-ukrainy-co-mo%C5%BCesz-zrobi%C4%87-na-allegro-aby-pom%C3%B3c/ba-p/334998) [PL],
+  provide means of communication, etc., depending on the industry.
+* If you work in tech, you are likely an authority on IT matters for your family and friends. Make them aware of
+  the issue of disinformation, and encourage them to use only reliable information sources. Due to the threat of
+  cyberattacks, encourage them to upgrade their systems and make backups.
+* You might also be able to [apply your tech skills directly](https://www.techtotherescue.org/tech/tech-for-ukraine).
```

Comment:
Do we want to mention cases of _DDoS_ attacks and websites that perform these kinds of attacks? We could warn about this not being legal in all countries and about programs or websites that can be malicious. One should take this type of action only if he knows what he's doing. Niebezpiecznik also [mentioned](https://twitter.com/niebezpiecznik/status/1497280631075655694) the fact that it actually can do nothing or be harmful to the attacker.

What do you think? It's quite controversial, though 🙂

Reply:
Yes, I considered this, but maybe it's better to not mention the topic at all since it gets quite complicated and I want to keep the text short. Even in IT world it's probably not the most popular way of engagement anyway.